newIDE/app/src/VariablesList/VariableRow.js

```diff
@@ -59,11 +63,28 @@ const VariableRow = ({
   onSelect,
   origin,
 }: Props) => {
-  const isStructure = variable.getType() === gd.Variable.Structure;
-  const key = '' + depth + name;
+  const type = variable.getType();
+  const isStructural =
```

Comment:
nitpicking: can we find another name for this?
"Structural type" does not mean what we mean here: https://en.wikipedia.org/wiki/Structural_type_system#:~:text=A%20structural%20type%20system%20(or,name%20or%20place%20of%20declaration.

I suggest "hasChildren" or "isCollection", with a preference for collection (in [Java](https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html) for example, it's the name to describe something that contains other things, including arrays, hash maps, sets).